### PR TITLE
chore: Remove strategies with space count 0 and last edited longer than 6 months

### PR DESCRIPTION
**Pull request by samuveth**
Closes: https://github.com/snapshot-labs/pitches/issues/26

List of removed strategies:

```
[
  'giveth-xdai-balance',
  'coordinape',
  'sunder',
  'balancer-erc20-internal-balance-of',
  'eth-received',
  'eth-philanthropy',
  'minto-balance-of-all',
  'protofi-erc721-tier-weighted',
  'synthetix-quadratic_2',
  'synthetix-non-quadratic',
  'synthetix-non-quadratic_2',
  'cream',
  'esd',
  'esd-delegation',
  'piedao',
  'ocean-marketplace-v4',
  'rebased',
  'ruler-staked-token',
  'avn-balance-of-staked',
  'iotex-balance',
  'squadz-power',
  'balancer-delegation',
  'tomyumswap',
  'planet-finance',
  'badgeth',
  'cronaswap',
  'snowswap',
  'hopr-staking',
  'swapr',
  'cyberkongz',
  'cyberkongz-v3',
  'hashflow-governance-power',
  'pod-leader',
  'vault-token-lp-balance',
  'single-staking-vault-balanceof',
  'svs-staking',
  'radicle-community-tokens',
  'digitalax-deco-to-mona',
  'blockzerolabs-cryptonauts',
  'minmax-mcn-farm',
  'jade-smrt',
  'saddle-finance',
  'saddle-finance-v2',
  'xkawa-farm',
  'sumami-holders',
  'zorro',
  'ethercats-founder-series',
  'gysr-pending-rewards',
  'safety-module-bpt-power',
  'capitaldao-staking',
  'erc20-rebase-wrapper',
  'uma-voting',
  'stakedao-governance-update',
  'hedgey-multi',
  'gin-finance',
  'aura-vlaura-vebal',
  'aura-vlaura-vebal-with-overrides',
  'aura-balance-of-vlaura-vebal',
  'solv-voucher-claimable',
  'apeswap',
  'dopamine',
  'lrc-nft-dao-search',
  'selfswap',
  'erc3525-vesting-voucher',
  'xrook-balance-of-underlying-weighted',
  'echelon-cached-erc1155-decay',
  'proxyprotocol-erc20-balance-of',
  'proxyprotocol-erc1155-balance-of',
  'riskharbor-underwriter',
  'clqdr-balance-with-lp',
  'dsla-parametric-staking-service-credits',
  'multichain-serie',
  'ctsi-staking',
  'vsta-pool-staking',
  'babywealthyclub',
  'echelon-wallet-prime-and-cached-key-gated'
]
```

### CODE REVIEW

**Comment by samuveth** [comment]
Ended up adding back some strategies because they were dependencies in other strategies.

**Comment by wa0x6e** [comment]
What's the query used to extract the list above ?

**Comment by wa0x6e** [comment]
Should we add a filter to only include hibernated spaces ? So we're not removing strategies for "active" spaces, and avoid breaking them.

This way, only truly inactive spaces will have invalid strategies, and they will be fixed on reactivation.

**Comment by samuveth** [comment]
@wa0x6e all strategies that were removed are used by zero spaces.

**Comment by wa0x6e** [comment]
> @wa0x6e all strategies that were removed are used by zero spaces.

So you just go there https://snapshot.org/#/?filter=strategies and remove strategies when it says "In 0 space(s)" ? Is there still a 6 months filter ?

**Comment by samuveth** [comment]
Added back the nested strategies you sent me in DM. @wa0x6e

**Comment by wa0x6e** [comment]
> Added back the nested strategies you sent me in DM. @wa0x6e

Can you update the list in your first post ?

**Comment by samuveth** [comment]
@wa0x6e updated the list

**Comment by wa0x6e** [comment]
```
'goldfinch-voting-power',
'goldfinch-membership',
```

are used by `goldfinch.eth`. The rest of the list of good

**Comment by samuveth** [comment]
@wa0x6e done

**Comment by samuveth** [comment]
@ChaituVR can you also make a quick review to be sure?

**Review by ChaituVR — APPROVED** [review]
tAck

sorry for delay 🙏

File: src/strategies/index.ts [hunk 1]
```diff
@@ -571,27 +460,15 @@ const strategies = {
...
-  'esd-delegation': esdDelegation,
-  piedao,
   'xdai-easy-staking': xDaiEasyStaking,
   'xdai-posdao-staking': xDaiPOSDAOStaking,
   'xdai-stake-holders': xDaiStakeHolders,
   'xdai-stake-delegation': xDaiStakeDelegation,
   defidollar,
   aavegotchi,
   'aavegotchi-agip': aavegotchiAgip,
-  'aavegotchi-agip-17': aavegotchiAgip17,
```
Comment (ChaituVR) [inline src/strategies/index.ts:594]: We removed a non-deleted strategy from here

Comment (ChaituVR) [reply src/strategies/index.ts:594]: Around 30 strategies are missing from this :(

Comment (ChaituVR) [reply src/strategies/index.ts:594]: added them here https://github.com/snapshot-labs/snapshot-strategies/pull/1366

**Pull request merged**